packages/sources/react/src/components/structure/VtmnDivider/VtmnDivider.tsx

```diff
@@ -37,7 +44,7 @@ export const VtmnDivider = ({
       role="separator"
       aria-orientation={orientation}
     >
-      {children}
+      <span id={String(props['aria-labelledby'])}>{children}</span>
```

Comment:
We need to add this attribute only if `ariaLabelId` is defined.

Reply:
Done, thanks :)